review-tools/gitaddrev

```diff
@@ -29,10 +29,22 @@ my $query = OpenSSL::Query->new();
 my @reviewers;
 my @nocla_reviewers;
 my @unknown_reviewers;
+my $min_reviewers = 2;          # All source default
+my $authorcount = 0;
+my $min_authors = 0;            # Main source default
```

Comment:
The name of this variable is slightly confusing as it is more like `count_authors` or `author_may_review`.

Reply:
Yes, a comment would be fine.

Reply:
Done